admin/pom.xml

```diff
@@ -98,9 +100,12 @@
                 <configuration>
                     <skip>false</skip>
                     <buildArgs>
-                        <version>${project.version}</version>
+                        <DOCKER_BASE_ARTIFACT>${dockerfile.base.artifact}</DOCKER_BASE_ARTIFACT>
```

Comment:
How about adding `<JAR_FILE>${project.build.finalName}.jar</JAR_FILE>` to build args and in Dockerfile have `ARG JAR_FILE` instead of `ARG VERSION` and on `COPY` you use that one.

This would make it also easy to build docker images outside of this mvn context.

Reply:
@wajda sure thing, if you find it problematic/unstable for your use case.